config/locales/en.yml

```diff
@@ -40,4 +40,8 @@ en:
         other: ">%{count}yr"
       almost_x_years:
         one:   "~1yr"
-        other: "~%{count}yr"
\ No newline at end of file
+        other: "~%{count}yr"
+  activerecord:
+    attributes:
+      agent:
+          receivers: Targets
```

Comment:
I went with `Reicevers`, thought about it a little and figured it does not really make a difference and prevents confusion like you said.